tests/test.ts

```diff
@@ -0,0 +1,32 @@
+import { JobEntrySubsystem } from "../index";
+
+const connectionOptions = {
+    host: "localhost",
+    password: "test",
+    user: "test",
+};
+
+const jobEntrySubsystem = new JobEntrySubsystem(connectionOptions);
+
+const JCLTestJob = `
```

Comment:
JCL actually looks pretty great in an ES6 template string, especially with your use of template tags.

Reply:
Doesn't it though :)